scripts/Game/Systems/Core/Managers/CRF_RplBroadcastManager.c

```diff
@@ -59,15 +59,32 @@ class CRF_RplBroadcastManager : ScriptComponent
 	}
 	
 	//------------------------------------------------------------------------------------------------
-	void SendAdminMessage(string data, int playerID)
+	void SendAdminMessage(string data, int playerID, bool ticketExists)
```

Comment:
There's a race condition issue here where `ticketExists` is calculated on the client, so if the client is slow, it's entirely possible this gets sent as false.

Recommend:
```
[RplRpc(RplChannel.Reliable, RplRcver.Server)]
protected void RpcAsk_SendAdminMessage(string data, int playerID)
{
    bool ticketExists = m_AdminMenuManager.TicketExists(playerID);
    m_RplBroadcastManager.SendAdminMessage(data, playerID, ticketExists);
    // ... rest of the logic
}
```